components/viaduct/android/src/main/java/mozilla/appservices/httpconfig/HttpConfig.kt

```diff
@@ -91,8 +91,7 @@ object RustHttpConfig {
                     }
                     rb
                 } catch (e: Throwable) {
-                    LibViaduct.INSTANCE.viaduct_log_error("Network error: ${e.message}")
-                    MsgTypes.Response.newBuilder().setExceptionMessage(e.message)
+                    MsgTypes.Response.newBuilder().setExceptionMessage("Network error: ${e.message}")
```

Comment:
This fixes the bug I described in https://mozilla.slack.com/archives/CBBH6GU91/p1593009194368100 and ends up in `Missing HTTP status` error in Sentry.